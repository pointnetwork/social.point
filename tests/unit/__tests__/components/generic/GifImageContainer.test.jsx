import React from "react";
import renderer from 'react-test-renderer';

import GifImageContainer from '../../../../../src/components/generic/GifImageContainer';

import image from "../../assets/header-pic.jpg"

describe("GifImageContainer", () => {

    it('should render', () => {
        const component = renderer.create(
            <GifImageContainer src={image}/>
        )
        expect(component.toJSON()).toMatchSnapshot();
    });

});