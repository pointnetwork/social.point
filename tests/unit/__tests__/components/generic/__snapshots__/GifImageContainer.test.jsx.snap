// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GifImageContainer should render 1`] = `
<div
  className="MuiBox-root MuiBox-root-4 makeStyles-root-1"
>
  <div
    className="makeStyles-frame-2"
  >
    <img
      alt=""
      className="makeStyles-image-3"
      src="test-file-stub"
    />
  </div>
</div>
`;
